### PR TITLE
Adding shortcode consolidation

### PR DESCRIPTION
**Pull request by MohanRaj168**
This PR only for bpr-v2 version

### CODE REVIEW

File: src/Shortcode/Shortcode.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            "render_business_profile",
+        ]);
+    }
+
+    // Method to check if bpr_business_profile option exists
+
+    public static function render_business_profile($attr)
+    {
+        // Parse JSON from wp_options
+        $json_data = get_option("bpr_business_profile");
```
Comment (rajanvijayan) [inline src/Shortcode/Shortcode.php]: Add `bpr_business_profile` as defined variable 

`define( 'BUSINESS_PROFILE_RENDER_OPTION', 'bpr_business_profile' );`

File: src/Shortcode/Shortcode.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+        if (!$json_data) {
+            return "Business profile data not found";
+        }
+
+        // Get the attribute value from shortcode or use default 'company_name'
+        $attr = isset($attr["attr"]) ? $attr["attr"] : "company_name";
```
Comment (rajanvijayan) [inline src/Shortcode/Shortcode.php]: Add company_name as defined variable

define( 'BUSINESS_PROFILE_RENDER_DEFAULT_OPTION', 'company_name' );

File: src/Shortcode/Shortcode.php [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+        if (!array_key_exists($attr, $json_data)) {
+            return "Attribute not found";
```
Comment (rajanvijayan) [inline src/Shortcode/Shortcode.php]: Make this string translatable 

[reference](https://codex.wordpress.org/I18n_for_WordPress_Developers)

**Review by rajanvijayan — CHANGES_REQUESTED** [review]

File: src/Shortcode/Shortcode.php (hunk 1 above)
Comment (MohanRaj168) [reply src/Shortcode/Shortcode.php]: Var defined in business-profile-render.php file

File: src/Shortcode/Shortcode.php (hunk 2 above)
Comment (MohanRaj168) [reply src/Shortcode/Shortcode.php]: Var defined in business-profile-render.php file

File: src/Shortcode/Shortcode.php (hunk 3 above)
Comment (MohanRaj168) [reply src/Shortcode/Shortcode.php]: updated.

**Pull request merged**